spec/models/post_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { User.create(name: 'John') }
+
+  it 'title valid with a title' do
+    first_user = User.first
+    post = Post.new(author: first_user, title: 'my first post', commentscounter: 0, likescounter: 0)
+    expect(post).to be_valid
+  end
```

Comment:
- [ ] This particular test case is failing, I got the error message screenshot below 👇 

<img width="543" alt="Untitled" src="https://github.com/devalibello/blog-app/assets/103386183/717ea100-3ea8-4da9-8504-734795dccaf0">